### PR TITLE
Update CHANGELOG for 2.27.0 release

### PR DESCRIPTION
**Pull request by shuoyang2016**
- Fix overhead latency calculation for backend timeout (#505)
- Reduce backoff initial latency. (#520)
- Update googleapis/googleapis for golang code (#516)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# Release 2.27.0 02-06-2021
+
+- b/184731851: Fix overhead latency calculation for backend timeout (#505)
+- b/188906398: Reduce backoff initial latency. (#520)
+- b/188808823: Simplify CORS regex and fix brittle tests (#519)
+- Update release scripts to allow patch release (#518)
+- b/188583118: Update googleapis/googleapis for golang code (#516)
+- Changelog for v2.26.1 (#517)
+- GCS Runner: Handle deadline exceeded properly (#515)
```
Comment (qiwzhang) [inline CHANGELOG.md:9]: remove lines that not user facing or feature or bug fixing.  such as. 8, 6, 9

Comment (shuoyang2016) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# Release 2.27.0 02-06-2021
+
+- b/184731851: Fix overhead latency calculation for backend timeout (#505)
+- b/188906398: Reduce backoff initial latency. (#520)
+- b/188808823: Simplify CORS regex and fix brittle tests (#519)
+- b/188583118: Update googleapis/googleapis for golang code (#516)
```
Comment (qiwzhang) [inline CHANGELOG.md:6]: Remove the bug ids.  They don't mean anything to external users.

Comment (shuoyang2016) [reply CHANGELOG.md]: Done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+# Release 2.27.0 02-06-2021
+
+- Fix overhead latency calculation for backend timeout (#505)
+- Reduce backoff initial latency. (#520)
+- Simplify CORS regex and fix brittle tests (#519)
```
Comment (nareddyt) [inline CHANGELOG.md:5]: remove line 5

**Review by qiwzhang — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (shuoyang2016) [reply CHANGELOG.md]: Done.

**Review by nareddyt — APPROVED** [review]

**Pull request merged**